optional_plugins/runner_docker/avocado_runner_docker/__init__.py

```diff
@@ -120,11 +120,11 @@ def __init__(self, job, result):
         self.remote = None      # Will be set in `setup`
 
     def setup(self):
-        dkrcmd = self.job.args.docker_cmd
-        dkr_opt = self.job.args.docker_options
+        dkrcmd = self.job.args.get('docker_cmd')
```

Comment:
At this point the ```job``` object hasn't ```args``` (rather ```config```), it has?

Reply:
Yes, it should have `.config` at this point.  Nice catch.